exercises/bowling/bowling_test.py

```diff
@@ -0,0 +1,261 @@
+import unittest
+
+from example import BowlingGame
+
+
+class BowlingTests(unittest.TestCase):
+    def test_should_be_able_to_score_a_game_with_all_zeros(self):
+        rolls = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+
+        self.game = BowlingGame()
```

Comment:
I would also suggest introduce a `setUp` method and create an instance of `BowlingGame` only once
```python
def setUp(self):
        self.game= BowlingGame()
```